src/tree.item.ts

```diff
@@ -111,8 +111,8 @@ export default class GlideCoreTreeItem extends LitElement {
   override render() {
     return html`<div
       aria-label=${ifDefined(this.label)}
-      aria-selected=${ifDefined(this.#ariaSelected)}
-      aria-expanded=${ifDefined(this.#ariaExpanded)}
+      aria-selected=${this.selected}
```

Comment:
> Tree Item is now presented to screenreaders as selected when selected

Reply:
I'm curious why this wouldn't have worked before, looking at the `#ariaSelected` getter...

But if the Item is not selectable, the attribute should be omitted completely, per https://www.w3.org/WAI/ARIA/apg/patterns/treeview/:

> If the tree contains nodes that are not selectable, neither aria-selected nor aria-checked is present on those nodes.

Reply:
I could be wrong. But I think it's because `this.#ariaSelected` isn't state and so isn't reactive. We have a number of cases like this, where something not marked as state is re-rendered only by virtue of the fact that something else that's marked as state  (`@state()`) changes at the same time.